docker/frontend-entrypoint.py

```diff
@@ -12,18 +12,21 @@ def main():
     frontend = os.environ.get('REACT_APP_FRONTEND_URI')
     auth = os.environ.get('REACT_APP_AUTH_MODE')
     use_ssl = os.environ.get('SSL') != 'false'
-    print("replacing", {"private": private, "public": public, "frontend": frontend})
+    print("replacing", {"private": private, "public": public, "frontend": frontend, "auth": auth})
 
     with open(CONSTANTS_FILE, 'r') as f:
         data = f.read()
         if auth:
             data = re.sub('firebase', auth, data)
         if private:
             data = re.sub('http://localhost:8082/private', private, data)
+            data = re.sub('https://pri\.highlight\.io', private, data)
         if public:
             data = re.sub('http://localhost:8082/public', public, data)
+            data = re.sub('https://pub\.highlight\.run', public, data)
```

Comment:
Should this be `.run` or `.io`? If `.run` is right, just curious why we are using that instead of `.io`?

Reply:
the value is `highlight.run` in our production docker image - we can probably update that at some point